optionals/compat_rhs_afrf3/CfgVehicles.hpp

```diff
@@ -282,14 +282,28 @@ class CfgVehicles {
     class Helicopter_Base_F;
     class Helicopter_Base_H: Helicopter_Base_F {
         class EventHandlers;
+        class Turrets;
+    };
+    class Heli_Light_02_base_F: Helicopter_Base_H {
+        class Turrets: Turrets {
+                class CopilotTurret;
+                class MainTurret;
```

Comment:
WRONG
You (or RHS) are introducing 
`CfgVehicles/Heli_Light_02_base_F/Turrets/MainTurret`
which does *not* exist in the base game.
The actual class is:
`CfgVehicles/Helicopter_Base_H/Turrets/MainTurret`
and you have to inherit it from `Helicopter_Base_H`, otherwise you are changing the details of class inheritance. Just because both classes are called `MainTurret` doesn't mean that they are the same.
This comes into play when you create a helicopter inheriting from `Heli_Light_02_base_F` without changing `Heli_Light_02_base_F`'s `Turrets` class. Since turrets are ignored when inherited without reference, in vanilla you would end up with one copilot turret, while with this addon you would end up with a gunner and a copilot. You created a bugged invisible (since most likely not supported by the model) ghost slot.